artman/cli/main.py

```diff
@@ -273,6 +274,10 @@ def normalize_flags(flags, user_config):
     elif artifact_type == Artifact.DISCOGAPIC_CONFIG:
         pipeline_name = 'DiscoGapicConfigPipeline'
         pipeline_args['discovery_doc'] = artifact_config.discovery_doc
+        if flags.output_dir != os.path.abspath(DEFAULT_OUTPUT_DIR):
```

Comment:
what if it's the same but just not the absolute path? maybe use `os.path.abspath(flags.output_dir)` as well to compare apples to apples?

Reply:
`output_dir` is already converted to absolute path here: https://github.com/googleapis/artman/pull/669/files/1a72f234a170a66b904b932488b8c801b5c5adad#diff-371ed57b95bd30de3b3331f9cc2ff446L217

But yeah, I see the point here. Adding the `abspath` here will make it less confusing.